backend/battling/urls.py

```diff
@@ -15,13 +15,14 @@
 
 urlpatterns = [
     path("", HomeView.as_view(), name="home"),
-    path("login/", LoginView.as_view(), name="login"),
-    path("logout/", LogoutView.as_view(next_page="login"), name="logout"),
-    path("signup/", SignupView.as_view(), name="signup"),
+    path("account/login/", LoginView.as_view(), name="login"),
+    path("account/logout/", LogoutView.as_view(next_page="login"), name="logout"),
+    path("account/signup/", SignupView.as_view(), name="signup"),
     path("battle/new/", CreateBattleView.as_view(), name="create_battle"),
     path("battle/list/", BattleListView.as_view(), name="battle_list"),
     path("team/<int:pk>/edit/", CreateTeamView.as_view(), name="create_team"),
     path("battle/<int:pk>/delete/", DeleteBattleView.as_view(), name="delete_battle"),
     path("battle/<int:pk>/detail/", DetailBattleView.as_view(), name="battle_detail"),
     path("oauth/", include("social_django.urls"), name="social"),
+    path("account/", include("django.contrib.auth.urls")),
```

Comment:
When you do that, the user will think they're resetting the password, when in fact they are setting it for the first time, so they'll be confused. You must redefine and have both behaviors: password reset and password set when first invited.

Reply:
You should inherit this view and replace the attrs that you need: https://github.com/django/django/blob/main/django/contrib/auth/views.py#L251-L259